dashboard/app/batch_main.go

```diff
@@ -17,7 +17,7 @@ import (
 
 func initBatchProcessors() {
 	http.HandleFunc("/cron/batch_coverage", handleBatchCoverage)
-	http.HandleFunc("/cron/batch_reproexport", handleBatchReproExport)
+	http.HandleFunc("/cron/batch_syzbot_export", handleBatchSyzbotExport)
```

Comment:
Everything in syzbot is syzbot-something. Using "syzbot" is excessive and inconsistent, we never use "syzbot" in the dashboard code anywhere else.
It would be more useful to say _what_ it's exporting, there are lots of potential things to export.

Reply:
s/syzbot/db/ ?

Reply:
"db" is fine if it exports all/most of the db.

Reply:
renamed